scripts/apps/search/directives/ItemList.js

```diff
@@ -1653,6 +1669,10 @@ export function ItemList(
                     var createItem = function createItem(itemId) {
                         var item = this.state.itemsById[itemId];
                         var task = item.task || {desk: null};
+                        var versionCreatorName;
```

Comment:
You can start off with `null` here and remove `|| null` on line 1691. Alternatively, as a better approach, you could create a function on this component called `displayName` which will return the display_name for a user, or null otherwise, and then call it as `this.displayName(item.version_creator)`.